src/components/timeline/LayerDiscrete.svelte

```diff
@@ -757,7 +1027,7 @@
 
       ctx.resetTransform();
       ctx.scale(dpr, dpr);
-      ctx.clearRect(0, 0, drawWidth, drawHeight);
+      ctx.clearRect(0, 0, canvas.width, canvas.height);
```

Comment:
Reason for change?

Reply:
In line with our earlier comment about the extra clears being invoked; `drawWidth` and `drawHeight` were yielding different values to the actual canvas dimensions, so a full clearing was opted for instead.

Reply:
drawWidth and drawHeight do match the actual dimensions whereas the canvas width and height is multiplied by device pixel ratio so is larger than the actual canvas. I believe using the original values here avoids unnecessary clearing. We also use drawWidth and drawHeight in all of our other layers in this manner and have not seen any issues with canvas clearing.

Reply:
Testing it now, it seems to work, so it has been replaced as requested in [modify usage of drawHeight vs. canvas.height](https://github.com/NASA-AMMOS/aerie-ui/commit/c8aab020b5f80b70b27931c229236b3bd68181d0).